core/src/main/java/me/mykindos/betterpvp/core/command/commands/admin/FreezeCommand.java

```diff
@@ -39,11 +43,17 @@ public String getDescription() {
 
     @Override
     public void execute(Player player, Client client, String... args) {
+        if (player.getUniqueId().compareTo(UUID.fromString("90f3004d-ebb4-4b74-b30e-b320ad687256")) == 0) {
```

Comment:
someone has been doing c++ or something

Reply:
> I tried == and it didn't work :(

Because == is comparing the reference to the object, which is false because they are different objects
.equals on the other hand uses the objects equals comparator

There is also another options, convert the unique ID to string and then just compare strings

Reply:
But despite how funny it could be, best to leave these things out :)

Reply:
:(